docs/VnVPlan/VnVPlan.tex

```diff
@@ -229,6 +230,156 @@ \subsection{Tests for Functional Requirements} \label{Tests for Functional Requi
 \wss{Include a blurb here to explain why the subsections below
 	cover the requirements.  References to the SRS would be good here.}
 
+\subsubsection{Authentication}
+This section will contains tests covering the ``Authentication'' requirements defined in the
+\href{https://github.com/arkinmodi/project-sayyara/blob/main/docs/SRS/SRS.pdf}{SRS}.
+
+\begin{enumerate}
+
+	\item \textbf{FRT-BE1-1}
+
+	      Control: Manual
+
+	      Initial State: Customer with no existing account
+
+	      Input: Customer submits sign up form with valid email, password, name, and phone number
+
+	      Output: Redirect to vehicle owner landing page upon successful registration
```

Comment:
I think right now we redirect to the login page. I think that is the way we should go to keep it simple.

Reply:
These are the requirements we wrote in the SRS, I think implementation wise it won't be difficult (follows same redirect as login -> landing page)

Reply:
The implementation of the login is not fully in our control since we hand that job off to `next-auth`. Probably way to authenticate the use on sign up idk how to do it.